pages/terms.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: page-no-toc
+title: "Terms of Use"
+permalink: terms
+---
+
+# Terms of Use
+
+## URI Persistence Policy
+
+Last modified: 2021-01-12
+
+When information is made available on the Web, it is important for the integrity of the Web, and the society based upon it, that the URIs used to reference information be used well into the future, and that the information persist as identified.
+
+To this end, the Solid Project hosts make the following pledge: that as far as they are able, for resources on the solidproject.org Web site which are declared (see below) to be persistent.
+
+Pledge:
+
+* The hosts will ensure that persistent resources continue to be available throughout the life of the Solid Project;
+* Where a persistent resource is modified, a change history will be archived though the archive will not necessarily be available publicly;
```

Comment:
Would the git history be archive enough, or should we look at other solutions ?

Reply:
FYI, I've updated the PR https://github.com/solid/solidproject.org/pull/489/commits/a03131ec30ebd8fe3f4c8bfa2388b6641bd7201e . Your question is still relevant:

For the time being, git history *in addition to* creating archived snapshots at third-party services (like the Internet Archive) - important - should suffice. Open for other solutions.

---
Perhaps TMI but FYI:

Using W3C's approach eg: `https://www.w3.org/TR/2017/REC-ldn-20170502/` is a Memento resource ( http://mementoweb.org/ns#Memento ) of a prior state of the Original Resource ( `https://www.w3.org/TR/ldn/` ). Note that we updated the original resource and the Memento resource also got updated. The change history is included in the "Status of the Document".

Compare first archived snapshot of the versioned resource: https://web.archive.org/web/20170514020225/https://www.w3.org/TR/2017/REC-ldn-20170502/ with the archived version of the versioned resource that got updated: https://web.archive.org/web/20171013165934/https://www.w3.org/TR/2017/REC-ldn-20170502/ re "Status of the Document".

This is possible through on-demand public archival services, and I would consider that it gives a basic and a reasonable coverage of the pledge. If we can do anything more advanced or preferable than the git history and the archival snapshot, let's have issues for that.

---

I'll follow-up parts that are relevant for the process in a different PR.